### DISCUSSION TITLE
MMTEB Author check

### DISCUSSION THREAD

**Discussion by KennethEnevoldsen** [embeddings-benchmark/mteb#1241] [General]
Hei all contributors, hope you are well,

We plan to submit it to ICLR in the coming days.

I have attached a (non-final) version of the paper here: 
~~[MMTEB-3.pdf](https://github.com/user-attachments/files/17151653/MMTEB-3.pdf)~~

Edit: updated version of the cover is here:

<img width="613" alt="Screenshot 2024-09-26 at 22 52 32" src="https://github.com/user-attachments/assets/fce3467b-1f57-443d-bf4c-4eac58d89bf7">

This post here is to allow authors to opt-out of authorship and/or allow authors to question ordering/authorship.

There are still a few final things that need to be added (especially template and anonymization), but if you find that there are specific points that need addressing let me know.

Tagging authors for reference:
@akshita-sukhlecha 
@rbroc 
@jupyterjazz 
@bp-high 
@Rysias 
@dwzhu-pku 
@gowitheflow-1998 
@x-tabdeveloping 
@simon-clematide 
@jphme 
@rafalposwiata 
@manandey 
@gentaiscool 
@henilp105 
@anpalmak2003 
@MathieuCiancone 
@rasdani 
@dipam7 
@awinml 
@artemsnegirev 
@shreeya-dhakal 
@Akash190104 
@ZhengLiu101 
@Art3mis07 
@taeminlee 
@GabrielSequeira 
@jaygala24 
@xiamengzhou 
@ljvmiranda921 
@Samoed 
@KennethEnevoldsen 
@digantamisra98 
@sivareddyg 
@mrshu 
@Ruqyai 
@asparius 
@staoxiao 
@MariyaTikhonova 
@isaac-chung 
@violenil 
@SaitejaUtpala 
@xhluca 
@ManuelFay 
@wissam-sib 
@bjoernpl 
@Alenush 
@Andrian0s 
@ABorghini 
@taidnguyen 
@cassanof

**Comment by KennethEnevoldsen** [comment]
The rest of the authors:

@Sakshamrzt 
@imenelydiaker 
@HLasse 
@guenthermi 
@MartinBernstorff 
@xu3kev 
@swj0419 
@mmhamdy 
@guangyusong 
@john-b-yang 
@slvnwhrl 
@PranjalChitale 
@malteos 
@davidstap 
@vaibhavad 
@AlexeyVatolin 
@KranthiGV 
@ShawonAshraf 
@jordiclive 
@jankounchained 
@ab1992ao 
@crystina-z 
@Muennighoff 
@hgissbkh 
@schmarion 
@dokato 
@mariyahendriksen 
@sted97 
@orionw 
@tomaarsen

**Comment by xhluca** [comment]
@KennethEnevoldsen It appears Mila appears twice, as 13 and 52
<img width="906" alt="image" src="https://github.com/user-attachments/assets/d5b45ef4-6c1f-4115-a8b0-f24ba6554547">

**Reply by HLasse** [reply]
and as 15

**Comment by isaac-chung** [comment]
ServiceNow Research shows up as 16 & 21

**Comment by davidstap** [comment]
Similar issue for University of Amsterdam; it shows up as 4 and 53.

**Comment by imenelydiaker** [comment]
Maybe we should standardize affiliations. Either have "Institution, Country" or just "Institution", wdyt @KennethEnevoldsen ?

**Reply by KennethEnevoldsen** [reply]
Institution is probably the simplest (no need to collect new information) - Will do this

**Reply by imenelydiaker** [reply]
Btw @KennethEnevoldsen, I removed my _Esker_ affiliation from the points.md file, I'm just keeping _INSA Lyon_

**Comment by SaitejaUtpala** [comment]
To find duplicates in the institutions listed, I'll go through the list and note any that appear more than once:

1. Stanford University - appears twice (entries 6 and 40)
2. Mila - Quebec AI Institute - appears three times (entries 13, 15, and 52)
3. ServiceNow Research - appears twice (entries 16 and 21)
4. University of Amsterdam - appears twice (entries 4 and 53)
5. SaluteDevices, Russia - appears twice (entries 46 and 54, though the second entry includes "HSE University")

These are the institutions that appear more than once in the list. All other institutions are listed only once.

**i asked claude, here are the duplicates**

**Comment by isaac-chung** [comment]
Some names (including mine) don't have superscripts. Could those authors be "individual contributors"?

**Reply by KennethEnevoldsen** [reply]
Added

**Comment by Sakshamrzt** [comment]
Please drop me ***@***.*** <https://github.com/Sakshamrzt>) from the list.

Thanks!

**Reply by KennethEnevoldsen** [reply]
Will do @Sakshamrzt, do you still want to be in the table of contributors or should I remove you there as well?

**Comment by KranthiGV** [comment]
Can we update the affiliation? If so, what's the process?
Should we raise a PR to update [points.md - contributor info](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md#contributor-informations)?

**Reply by isaac-chung** [reply]
Might just be easier to make a comment here and one of us can make the edits in the Overleaf file directly. What would you like to change it to?

**Reply by KennethEnevoldsen** [reply]
yes as @isaac-chung says

**Comment by isaac-chung** [comment]
I'll go through the draft to remove the duplicates.

**Reply by KennethEnevoldsen** [reply]
I will probably do this in a script (easier to do with the other changes as well)

**Reply by isaac-chung** [reply]
@KennethEnevoldsen feel free to override my changes as you see fit.

**Comment by crystina-z** [comment]
My name (Xinyu Zhang) seems to duplicate twice while @thakur-nandan (Nandan Thakur) name is not included in the author list

**Reply by KennethEnevoldsen** [reply]
Thanks!! Would have hated to miss that. I assume it is:

first name: Nandur
last name: Thakur
Aff: University of Waterloo
id on openreview: ~Nandan_Thakur1

(but do correct me if I am wrong)

**Reply by thakur-nandan** [reply]
That's correct. Sorry I just checked this.

**Comment by KennethEnevoldsen** [comment]
I have now corrected the duplicates, missing authors, added "individual contributor" for non-aff authors and removed @Sakshamrzt 

(I have added an updated screenshot in the top)

**Comment by cassanof** [comment]
Hi! Thank you so much for including me in the list of contributors.
Would it be possible to change my affiliation to "Northeastern University and Cursor AI"? 

Thanks again!

**Reply by imenelydiaker** [reply]
Please update this file and open a pull request: https://github.com/embeddings-benchmark/mteb/blob/main/docs%2Fmmteb%2Fpoints.md

**Reply by KennethEnevoldsen** [reply]
fixed !

**Comment by digantamisra98** [comment]
@KennethEnevoldsen Can my affiliation please be updated to: Max Planck Institute for Intelligent Systems and ELLIS Institute Tübingen. Thank you!

**Reply by imenelydiaker** [reply]
Please update this file and open a pull request: https://github.com/embeddings-benchmark/mteb/blob/main/docs%2Fmmteb%2Fpoints.md

**Reply by KennethEnevoldsen** [reply]
Fixed!

**Reply by digantamisra98** [reply]
https://github.com/embeddings-benchmark/mteb/pull/1247
Done

**Comment by dwzhu-pku** [comment]
Thank you so much for including me in the list of contributors! @KennethEnevoldsen

**Comment by jhrystrom** [comment]
Great work, Kenneth! On the screenshot, your email address seems to be missing a letter:))

**Reply by KennethEnevoldsen** [reply]
fixed ;)

**Comment by kwojtasi** [comment]
@KennethEnevoldsen Is it possible to be included in the author's list? I have contributed BEIR-PL benchmarch for Polish language for MTEB benchmark.

**Reply by KennethEnevoldsen** [reply]
Of course @kwojtasi. I went through the PR that would result in 22 points (from dataset submission). I have not counted any bonuses here, but it does justify for an authorship.

Can you (just here) write your author first, last name, affiliation and most importantly your openreview ID.

**Reply by kwojtasi** [reply]
Thank you! 
Providing the details:
Konrad Wojtasik, Wrocław University of Science and Technology, ~Konrad_Wojtasik1

**Comment by digantamisra98** [comment]
Who's registering the abstract? The deadline is today EOD I believe. @KennethEnevoldsen

**Reply by KennethEnevoldsen** [reply]
I am - doing it now

**Reply by digantamisra98** [reply]
Thank you!

**Comment by dokato** [comment]
Anything we could do to help with addressing reviewers' comments @KennethEnevoldsen ?

**Reply by Samoed** [reply]
It would be great if you could help close https://github.com/embeddings-benchmark/mteb/issues/1475. The reviewers are requesting statistics for the task, and with descriptive stats, these can be calculated automatically.

**Reply by isaac-chung** [reply]
I think @imenelydiaker's supporting that. Pinging her to see if she needs any additional support.

**Reply by KennethEnevoldsen** [reply]
Thanks @dokato I believe most points are being addressed. One point which might be ideal is the comparison of MTEB(classic) and MTEB(eng). Where we want to compare the runtime in a comparable manner (currently we are reporting runtime on A100 on one and H100 on the other), a solution to this is the just report the number of characters to embed pr. benchmark.

I could imagine the code could look something like:

``` 
bench = mteb.get_benchmark("mteb(eng, beta)")
n_char = 0 

for task in bench.tasks:
  n_char = task.metadata.descriptive_stats["number of characters"]
```

And then update the abstract intro and sections where we discuss runtime/size

**Reply by SaitejaUtpala** [reply]
@KennethEnevoldsen  Just want to say I love newly added Figure 4 !!

**Reply by KennethEnevoldsen** [reply]
Great to hear! @x-tabdeveloping is the man behind it, but I agree - it quite reasonably reveals the current language gap even for reasonably mid to high-resource languages.

**Reply by x-tabdeveloping** [reply]
Haha thanks 😅🥰

**Comment by manandey** [comment]
@KennethEnevoldsen any plans on having the paper on arxiv?

**Reply by KennethEnevoldsen** [reply]
yes! We plan to push it as soon as the next version of the leaderboard is ready (start of january). Hopefully that works?

**Reply by KennethEnevoldsen** [reply]
(i believe it is also public [here](https://openreview.net/forum?id=zl3pfz4VCV))

**Reply by manandey** [reply]
Great, thanks!

**Reply by isaac-chung** [reply]
Link is live: https://arxiv.org/abs/2502.13595v1

**Comment by wissam-sib** [comment]
:tada: :tada: :tada:

**Comment by mmhamdy** [comment]
Hello! Congratulations everyone🎉, any updates about the paper?

**Reply by isaac-chung** [reply]
Hey! As @KennethEnevoldsen mentioned in [this comment](https://github.com/embeddings-benchmark/mteb/discussions/1241#discussioncomment-11664116), we'd like to release the paper along with the new leaderboard. [This is the latest overview issue](https://github.com/embeddings-benchmark/mteb/issues/1867) on its development. We'd appreciate some help to get it closed sooner! Ping @x-tabdeveloping to see where you can contribute 🤗

**Reply by mmhamdy** [reply]
Hi! Thanks @isaac-chung for sharing this overview! 🙏 will have a look at it.

**Comment by sam-hey** [comment]
Hi @KennethEnevoldsen,

I’m fully aware that I’m likely out of time to be included as an author, but I wanted to ask if it might still be possible to be listed as an author on the arXiv paper. I contributed in several ways:

   - Implemented ColBERT and added 2 models
   - Reviewed multiple PRs
   - Fixed numerous issues
   - Introduced new features for MTEB v2
  - Created the German version of the leaderboard
   - Added over 200 results to the repository

Let me know what you think, and I completely understand if the timing doesn't allow for it.

Thanks!

**Reply by isaac-chung** [reply]
Hey @sam-hey,

First of all, we thank you for all your contributions so far! They certainly appreciated in the repository that is used by the community. 

However, your contributions were not in the MMTEB paper, so it would not be possible to add you as a author. For reference, [here is the folder where the MMTEB contributions were documented](https://github.com/embeddings-benchmark/mteb/tree/main/docs/mmteb). We based on that to determine author-ship. 

That said, if we open up contributions for a different paper in the future, we will be sure to announce it and hope you will join!
